migrations/sql/V2019.03.25.11.06__add_hsrc_code.sql

```diff
@@ -0,0 +1,1051 @@
+CREATE TABLE IF NOT EXISTS article_act_code
+(
+    article_act_code character varying(5)                                NOT NULL,
+    description      character varying(100)                              NOT NULL,
+    effective_date   date                     DEFAULT now()              NOT NULL,
+    expiry_date      date                     DEFAULT '9999-12-31'::date NOT NULL,
+    update_timestamp timestamp with time zone DEFAULT now()              NOT NULL
+);
+
+COMMENT ON TABLE article_act_code IS 'Contains a list of compliance sections as defined in a Legislative Act or Regulation; i.e. HSRC 1.2.2';
+
+ALTER TABLE ONLY article_act_code
+    ADD CONSTRAINT article_act_code_unique UNIQUE (article_act_code);
+
+INSERT INTO article_act_code
+(
+    article_act_code,
+    description,
+    effective_date,
```

Comment:
We should add a comment here too (because effective/expiry dates have been confusing in the past)... description could be "Effective Date the date the Act or Regulation is legally enacted"